sql/sttm-api issue15 - wordcount_arrays.sql

```diff
@@ -0,0 +1,117 @@
+create or replace view v_Ceremonies_Shabad as
```

Comment:
what's with this file name. are these like liquibase scripts to make db changes as we evolve?

Reply:
basically not having documented the history of the ddl well has caused issues, so now when I make ddl changes I want them committed to repo so its documented/traceable